Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -226,6 +226,66 @@ noncomputable def fieldEquivOfRingEquiv [Algebra B L] [IsFractionRing B L] (h :
         mem_nonZeroDivisors_iff_ne_zero, mem_nonZeroDivisors_iff_ne_zero]
       exact h.symm.map_ne_zero_iff)
 
+@[simp]
+lemma fieldEquivOfRingEquiv_algebraMap [Algebra B L] [IsFractionRing B L] (h : A ≃+* B)
+    (a : A) : fieldEquivOfRingEquiv h (algebraMap A K a) = algebraMap B L (h a) := by
+  simp [fieldEquivOfRingEquiv]
+
+section fieldEquivOfAlgEquiv
+
+variable {A B C : Type*}
+  [CommRing A] [CommRing B] [CommRing C]
+  [IsDomain A] [IsDomain B] [IsDomain C]
+  [Algebra A B] [Algebra A C]
+  (K L M : Type*) [Field K] [Field L] [Field M]
+  [Algebra A K] [Algebra B L] [Algebra C M]
+  [IsFractionRing A K] [IsFractionRing B L] [IsFractionRing C M]
+  [Algebra A L] [IsScalarTower A B L]
+  [Algebra A M] [IsScalarTower A C M]
+  [Algebra K L] [IsScalarTower A K L]
+  [Algebra K M] [IsScalarTower A K M]
+
+/-- An algebra isomorphism of rings induces an algebra isomorphism of fraction fields. -/
+noncomputable def fieldEquivOfAlgEquiv (f : B ≃ₐ[A] C) : L ≃ₐ[K] M where
+  __ := IsFractionRing.fieldEquivOfRingEquiv f.toRingEquiv
+  commutes' x := by
+    obtain ⟨x, y, -, rfl⟩ := IsFractionRing.div_surjective (A := A) x
+    simp_rw [map_div₀, ← IsScalarTower.algebraMap_apply, IsScalarTower.algebraMap_apply A B L]
+    simp [← IsScalarTower.algebraMap_apply A C M]
+
+@[simp]
+lemma fieldEquivOfAlgEquiv_algebraMap (f : B ≃ₐ[A] C) (b : B) :
```

Comment:
Can you add a comment explaining the relationship between this and `(fieldEquivOfAlgEquiv f).commutes` (ie, the rings involved), or maybe draw some commutative diagram if you think that is appropriat ehere?